src/config.c

```diff
@@ -2544,7 +2544,7 @@ standardConfig configs[] = {
     createLongLongConfig("slowlog-log-slower-than", NULL, MODIFIABLE_CONFIG, -1, LLONG_MAX, server.slowlog_log_slower_than, 10000, INTEGER_CONFIG, NULL, NULL),
     createLongLongConfig("latency-monitor-threshold", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.latency_monitor_threshold, 0, INTEGER_CONFIG, NULL, NULL),
     createLongLongConfig("proto-max-bulk-len", NULL, MODIFIABLE_CONFIG, 1024*1024, LONG_MAX, server.proto_max_bulk_len, 512ll*1024*1024, MEMORY_CONFIG, NULL, NULL), /* Bulk request max size */
-    createLongLongConfig("stream-node-max-entries", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX/100, server.stream_node_max_entries, 100, INTEGER_CONFIG, NULL, NULL),
+    createLongLongConfig("stream-node-max-entries", NULL, MODIFIABLE_CONFIG, 1, LONG_MAX, server.stream_node_max_entries, 100, INTEGER_CONFIG, NULL, NULL),
```

Comment:
now you're not permitting it to be set to 0, we certainly don't want that.
first, that's a documented feature (see redis.conf) which this change will mess up.
there's also explicit code that handles the case when stream_node_max_entries is 0, so even it it wasn't documented, it is intended.
and even if it wasn't, this would be a breaking change that will cause server to fail to start on upgrade (if user used 0 and did CONFIG REWRITE).

regarding LONG_MAX vs LLONG_MAX, i don't care much, i don't expect people to ever use high values, and even if they will, the implications are not that severe (they'll suffer more from non-performant streams, rather than the trimming not working)